### DISCUSSION TITLE
Compatibility with Debian 11 / i3WM?

### DISCUSSION THREAD

**Discussion by OposDev** [Almamu/linux-wallpaperengine#112] [Q&A]
Does this work with Debian 11 with i3WM? I would love to use this if it works properly :D

**Comment by OposDev** [comment]
To add on, how is the performance with this app?

**Comment by Almamu** [comment]
If you have i3 it should work just fine as long as something is not drawing on the background, take in mind that there's a lot of different features that we do not support yet, so most wallpapers will have issues or not properly display.
